drivers/SmartThings/zwave-siren/src/init.lua

```diff
@@ -68,6 +68,14 @@ local function do_configure(driver, device)
   end
 end
 
+--- Handle device added
+---
+--- @param self st.zwave.Driver
+--- @param device st.zwave.Device
+local function added_handler(self, device)
+  capabilities.defaults.emit_default_events(device, self.supported_capabilities)
```

Comment:
My only concern here is if we want the default to be to do this for _all_ the capabilities or if we want to be more specific to the problem events?  Also if any of the sub-drivers have their own `added` handlers this won't happen for them so we should review them to determine if this needs to be added to any of their `added` handlers as well.

Reply:
I feel pretty confident we don't lose anything by emitting it for everything